test/controllers/user.test.ts

```diff
@@ -32,4 +32,25 @@ describe('User class', () => {
             ).toThrowError();
         });
     });
+    describe('generateKeywords()', () => {
+        let user: User;
+
+        beforeEach(() => {
+            user = new User(new JsKeyManager(), jsStorageFactory);
+        });
+
+        it('generates random passphrase words', async () => {
+            const generatedKeywords = await user.generateRandomPassphrase();
+
+            expect(generatedKeywords).toHaveLength(6);
+        });
+
+        it('generates suggested passphrase words', async () => {
```

Comment:
Some more tests, what happens if user sends empty string. What happens if user sends a word that is not in the list. What happens if a user sends some text that does not have any results?